src/Stratis.Bitcoin.IntegrationTests/Mempool/MemoryPoolTests.cs

```diff
@@ -205,17 +189,15 @@ public void TxMempoolMapOrphans()
         {
             var rand = new Random();
             var randByte = new byte[32];
-            Func<uint256> randHash = () =>
+            uint256 randHash()
```

Comment:
funky

Reply:
Yeah this was suggested by visual studio ;)  to use a local function